client/src/app/pages/external/jira/trackers.tsx

```diff
@@ -259,11 +266,15 @@ export const JiraTrackers: React.FC = () => {
                           width={10}
                           {...getTdProps({ columnKey: "connection" })}
                         >
-                          <TrackerStatus
-                            name={tracker.name}
-                            connected={tracker.connected}
-                            message={tracker.message}
-                          />
+                          {updatingTrackerId === tracker.id ? (
```

Comment:
This will be "the most recently updated/created tracker".  Might want to track an array of tracker ids just to be sure that you get a spinner on multiple trackers if they're both edited very close together.  

Maybe a Set would work better?  But then since a Set is mutable, getting `useState()` to track changes would be a bit of a challenge.  Some stackoverflow discussion: https://stackoverflow.com/questions/58806883/how-to-use-set-with-reacts-usestate

Reply:
Converted to use a set internal to the hook but exported the readonly state as an array. Interesting on the mutable workaround. Updated the hook to use the approach you linked.